src/lib/layout/defaultActionLibrary.ts

```diff
@@ -0,0 +1,69 @@
+import { refreshPage } from '@src/lib/utils'
+import { reportRejection } from '@src/lib/trap'
+import { commandBarActor, settingsActor } from '@src/lib/singletons'
+import { useContext } from 'react'
+import { MachineManagerContext } from '@src/components/MachineManagerProvider'
+import { isDesktop } from '@src/lib/isDesktop'
+import { useReliesOnEngine } from '@src/hooks/useReliesOnEngine'
+import type { ActionType, ActionTypeDefinition } from '@src/lib/layout/types'
+
+/**
+ * For now we have strict action types but in future
+ * we should make it possible to register your own in an extension.
+ */
```

Comment:
Sick! Open in VS Code or like Send STL to PrusaSlicer would be easy ones to start with 🔥